cmd.go

```diff
@@ -26,3 +29,13 @@ func (c CmdLinux) ListContent(path string) ([]string, error) {
 
 	return lsOutput, nil
 }
+
+func (c CmdLinux) IsDir(dir string) bool {
```

Comment:
`IsDir()` is a generic function, could you remove it from the struct CmdLinux and implement as a normal helper function?